src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -291,15 +81,15 @@ private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, IEnumera
                 var candidateFullName = assembly.FullName;
                 if (String.Equals(candidateFullName, searchForFullName, StringComparison.OrdinalIgnoreCase))
                 {
-                    return assembly;                    
+                    return assembly;
                 }
             }
             return null;
         }
 
         private static Assembly MatchWithLoadedAssembly(AssemblyName searchFor, AppDomain appDomain)
         {
-            return 
+            return
```

Comment:
@xiazen You are correct. This thing shouldn't be used anymore. I'll get back to it later on. Thanks